pxtrunner/runner.ts

```diff
@@ -597,6 +597,10 @@ ${files["main.ts"]}
                     steps = tutorialmd.split(/^###[^#].*$/gmi);
                     newAuthoring = false;
                 }
+                if (steps[0].indexOf("# Not found") == 0) {
+                    pxt.log(`Tutorial not found: ${tutorialid}`);
+                    throw new Error(`Tutorial not found: ${tutorialid}`);
```

Comment:
use ``lf`` if message displayed to user

Reply:
Yep. thanks

Reply:
Message not displayed to the user. Just goes to our logs